website/src/components/CustomBanner/StandWithUkraine.module.css

```diff
@@ -2,7 +2,7 @@
   display: flex;
   justify-content: center;
   align-items: center;
-  background-color: #1f243c;
+  background-color: #1f253c;
```

Comment:
Did they also ask to change this color?

Reply:
They included it in the figma image for style matching. When inspecting, this is the hex showing - very minor but it does seem to be just a bit darker IMO. I can revert it if you prefer the previous. Please let me know!
![2022-12-14_15-00-40](https://user-images.githubusercontent.com/34761997/207602374-04251568-721b-4f28-a98c-71f0d00e8533.jpg)

Reply:
Dont revert, that's great :+1: